content/handbook/people-policies/france-sas/right-to-disconnect-charter.md

```
@@ -0,0 +1,395 @@
+---
+title: "GitLab France S.A.S. つながらない権利憲章"
+description: GitLab France S.A.S. つながらない権利憲章 - フランス語と英語のバージョン。
+upstream_path: /handbook/people-policies/france-sas/right-to-disconnect-charter/
+upstream_sha: 1e195b58b9f249ff10bd0e705106c320fee86141
+translated_at: "2026-05-14T00:00:00Z"
+translator: claude
+stale: false
+---
+
+{{% panel header="**これは Secure Document です**" header-bg="orange" %}}
+記載された[Roles & Responsibilities](#roles-and-responsibilities)のとおり、このページへの変更は code owner による承認またはマージが必要です。すべての貢献を歓迎し、奨励します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)